src/handlers/Registry.ts

```diff
@@ -182,12 +184,14 @@ export async function handleNewResolver({
         domainId: event.args.node,
         address: event.args.resolver,
       })
-      .onConflictDoNothing();
+      .onConflictDoUpdate({}); // noop update to return the existing record
```

Comment:
I believe that `onConflictDoNothing()` also returns the existing record:
- https://ponder.sh/docs/indexing/write-to-the-database#upsert--conflict-resolution

Perhaps `onConflictDoNothing()` and `onConflictDoUpdate({})` have the same effect?

Reply:
Ok, I've checked with the Ponder crew. If we need to:
- insert a DB entity if it does not exists, AND
- do nothing with it when it already exists, AND
- return the DB entity

Then we need `onConflictUpdate({})`. [More context on that issue](https://t.me/ponder_sh/9841):
> on conflict do nothing does not return anything when nothing is done. this is how Postgres does it. it is certainly a bit unintuitive. see this discussion for some further information on this functionality in postgres and workarounds https://stackoverflow.com/questions/34708509/how-to-use-returning-with-on-conflict-in-postgresql